docs/project/contribute.md

```diff
@@ -64,13 +64,56 @@ Search is by algolia, the configuration is at <https://github.com/algolia/docsea
 
 ## Running the site locally:
 
-All Contributors uses Docusaurus to maintain our documentation website. For more info visit the [Docusaurus Website](https://docusaurus.io)
+All Contributors currently uses [Docusaurus 1.x](https://docusaurus.io) for our documentation website.
 
-To get started locally:
+To build the docs locally:
 
-- `yarn install`
-- `yarn start`
-- go to `http://localhost:3000` (Refresh the page for your changes to be reflected)
+* <https://tutorial.docusaurus.io/docs/intro>
+
+1. Make sure `npm` is installed on your machine. Use the [nodejs documentation](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) if you aren't sure how to install `npm`.
+
+To check that npm is installed run the following in your favorite shell:
+
+`npm -v`
+
+Next, make sure you have at least node version 0.22 (LTS) or higher. To check the version,
+use:
+
+`node -v`
+
+If you don't have at least version `0.22` or higher, please install Node or upgrade your current version.
+
+Install docusaurus v1 (1.14.7). *Note: This project does not yet support docusaurus v2 or v3; though, we hope to do so in the future.*
+
+`npm install docusaurus@1.14.7 --save-dev`
```

Comment:
Maybe we should add docusaurus to the dependencies so we can just run `npm install`?

Reply:
So I just looked - it's there - i'll update it and you are correct, we can then just use npm install. Thanks for catching that!